packages/commonwealth/client/styles/components/component_kit/new_designs/CWSearchBar.scss

```diff
@@ -0,0 +1,215 @@
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 8px 16px;
+  gap: 8px;
+  height: 40px;
+  background-color: $white;
+  border: 1px solid $neutral-200;
+  border-radius: 6px;
+  color: $neutral-800;
+
+  &:focus-within {
+    outline: none;
+    border-color: $primary-500;
+    box-shadow: 0px 2px 2px -1px rgba(0, 0, 0, 0.12),
+    0px 0px 0px 3px $primary-100;
+  }
+
+  &.container {
+    position: relative;
+    padding: 0;
+    border: none;
+
+    &:hover {
+      border: none;
+    }
+  }
+
+  &.disabled {
+    background-color: $neutral-50;
+    border-color: $neutral-200;
+    box-shadow: none;
+    color: $neutral-500;
+
+    &:hover {
+      border-color: $neutral-200;
+    }
+  }
+
+  &:hover {
+    border: 1px solid $primary-600;
+  }
+
+  input {
+    width: 100%;
+    border: none;
+
+    &:focus {
+      outline: none;
+      box-shadow: none;
+    }
+
+    &.disabled {
+      background-color: $neutral-50;
+    }
+
+    ::placeholder {
+      color: $neutral-800;
+    }
+  }
+
+  &.magnifyingGlass {
+    color: $neutral-400;
+    width: fit-content;
+    padding: 0;
+    height: fit-content;
+    background-color: inherit;
+    border: none;
+    border-radius: 0;
+
+    &:hover {
+      cursor: pointer;
+      color: $neutral-600;
+    }
+  }
+  
+  &.inputElement {
+    flex: 1;
+    width: fit-content;
+    padding: 0;
+    height: fit-content;
+    background-color: inherit;
+    border: none;
+    box-shadow: none;
+    border-radius: 0;
+  }
+}
+
+.ListBox {
+  position: absolute;
+  top: calc(100% + 4px);
+  left: 0;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 6px;
+  margin-top: 0px;
+  max-height: 300px;
+  border-radius: 6px;
+  overflow: auto;
+  outline: 0px;
+  z-index: 1;
+  background-color: $white;
+  border: 1px solid $neutral-200;
+  color: $neutral-900;
+  box-shadow: 0px 4px 30px $neutral-200;
+
+  .previews-section {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    padding: 16px;
+
+    .preview-section {
+      display: flex;
+      flex-direction: column;
+
+      .section-header {
+        align-items: center;
+        display: flex;
+        gap: 8px;
+        margin-bottom: 8px;
+
+        .section-header-text.Text {
+          color: $neutral-500;
+        }
+      }
+    }
+  }
+
+  .no-results {
+    display: flex;
+    align-items: center;
+    padding: 4px;
+  }
+}
+
+.SearchBarThreadPreviewRow {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+
+  @include everyRowStyles;
+
+  .header-row {
+    align-items: center;
+    display: flex;
+    gap: 8px;
+
+    .User {
+      align-items: center;
+      display: flex;
+
+      .user-avatar {
+        position: unset;
+      }
+
+      .username {
+        @include caption;
+
+        font-weight: 500;
+      }
+    }
+
+    .last-updated-text.Text {
+      color: $neutral-500;
+    }
+  }
+
+  .excerpt-text.Text {
+    color: $neutral-600;
+  }
+}
+
+.SearchBarCommentPreviewRow {
```

Comment:
look at my other comment about having separate css files for each component

Reply:
Addressed.